content/docs/lists-and-keys.md

```diff
@@ -119,7 +119,7 @@ const todoItems = todos.map((todo) =>
 );
 ```
 
-When you don't have stable IDs for rendered items, you may use the item index as a key as a last resort:
+Cuando no tengas IDs estables para renderizar, podrias usar el indice del item como una key como ultimo recurso:
```

Comment:
I'd change `podrías` by `puedes` or `es posible`.
By the way, note the `í` on `podrías` and `ú` in `último`.